web/application/views/airqo-career-details1.php

```diff
@@ -0,0 +1,49 @@
+	<section class="w3l-about1">
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+	</section>
+	<section class="w3l-about2">
+		<div class="features-main py-5">
+			<div class="container py-lg-3">
+				<h3 class="stat-title">Digital Communications Officer <small> (Deadline: 11th May 2021 EAT)</small></h3>
+			</div>
+		</div>
+	</section>
+
+	<section class="w3l-about4">
+		<div class="contact1-bg py-0">
+			<div class="container py-lg-3">
+				<div class="contact-main row align-items-center">
+					<div class="col-md-12">
+						<div class="column">
+							<h3 class="header mb-4">Job Details</h3>
+							<p class="head-main mb-3">
+								Are you passionate about new media, have a critical eye for a great story and can communicate complex ideas about science, technology and social issues in innovative ways to diverse audiences? We would love to hear from you.
```

Comment:
In the near future, such updates should not be manual, we should explore options that empower non tech users to update such content on the site. It is a high priority feature.